### ISSUE TITLE
Import number started with 0 as string.

### ISSUE THREAD

**Issue by rafaelsisweb** [mongodb/mongo-tools#34]
Hi, we have a situation where our csv file has a field containing "010101" using mongoimport this value is parsed as Integer so what we get in mongocollection is "**10101** instead original **010101**. We had custom change on older version of mongo import (C++).
```
src/mongo/tools/import.cpp

void _append( BSONObjBuilder& b , const string& fieldName , const string& data ) {
        if ( _ignoreBlanks && data.size() == 0 )
            return;

        if (data[0] != '0' && _type == CSV) {
        if ( b.appendAsNumber( fieldName , data ) )
                return;
    }
```

It's possible to something like that on this GO version?

**Comment by 3rf** [comment]
Hi Rafael,

If you would like to duplicate your custom change from the C++ version in the new Go tools, I suggest you take a look at `getParsedValue` in https://github.com/mongodb/mongo-tools/blob/1fdcb1938ad307bf93599bba7086518a2b09fbd1/mongoimport/common.go#L148

You should be able to trivially add a check there and return a string if `token[0] == '0'`

Thanks,
Kyle

**Comment by mpobrien** [comment]
Hi @rafaelsisweb - we've recently added a feature that allows typed fields for CSV and TSV import, which I believe can be used to resolve your situation. See https://jira.mongodb.org/browse/TOOLS-67.
It's available in master now, and will become available in the next stable release (3.4).
